package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
Just checked--it builds and runs without any peer dependencies. Additionally, the fix that [cordova-plugin-file-transfer](https://github.com/apache/cordova-plugin-file-transfer/blob/master/package.json) made was to remove all of their peer dependencies.